client/app/components/composites/ListingCard/ListingCard.css

```diff
@@ -17,6 +17,8 @@
 /* Firefox doesn't support image aspect ratio inside flexbox */
 .aspectWrapper {
   padding-bottom: 100%;
+  border-top-left-radius: inherit;
+  border-top-right-radius: inherit;
```

Comment:
This is to fix a bug. 
(Introduced by this firefox fix - border-radius info was not passed through.)